requirements-dev-py2.txt

```diff
@@ -0,0 +1,12 @@
+numpydoc >= 0.6.0
```

Comment:
why is it necessary to have a second requirements file? are the dependencies different?